packages/autocomplete-core/src/createAutocomplete.ts

```diff
@@ -69,11 +69,11 @@ export function createAutocomplete<
   }
 
   if (
-    options.insights &&
+    props.insights &&
```

Comment:
We were previously using the passed option, not the defaulted one—this changes it.

Reply:
you could also leave this as `options`, and change `getDefaultProps`

Reply:
I find it cleaner to always rely on `props` as there's no ambiguity about what we're manipulating.

Reply:
Somewhat inconsistent with the other parts of the code which rely on options, but ok.